.gitlab-ci.yml

```diff
@@ -1079,6 +1095,36 @@ workflow:
   - when: manual
     allow_failure: true
 
+.on_windows_installer_changes_or_manual:
```

Comment:
`.on_windows_installer_changes_or_manual` is also defined on line 992, so this redefines it. Could you remove the obsolete definition?

Also, I'm not seeing changes in this new definition, except for the `- <<: *if_windows_installer_changes` block being replaced by its explicit definition. Is this intended?

Reply:
It was a mistake when I merged in the latest changes from `main`, I fixed it and removed the duplicate key, now it should be ok, sorry.